tasks/partial_bundle.js

```diff
@@ -35,8 +37,9 @@ module.exports = function partialBundle(tasks, opts) {
                 var newCode = partialIndex.replace(
                     new RegExp(
                         WHITESPACE_BEFORE +
-                        'require\\(\'\\./' + t + '\'\\),',
-                    'g'), ''
+                        'require\\(\'\\./' + t + '\'\\),' + '|' +
+                        'require\\(\'\\.\\./src/traces/' + t + '/strict\'\\),',
+                        'g'), ''
```

Comment:
Thanks very much for the PR.
Could you possibly add this extra regex condition only for `strict` bundles?